prebuild.js

```diff
@@ -25,14 +25,14 @@ function prebuild (opts, target, callback) {
       }
       var tasks = [
         function (cb) {
-          build(opts, target, function (err, filename) {
+          build(opts, target, function (err, filenames) {
             if (err) return cb(err)
-            cb(null, filename)
+            cb(null, filenames)
           })
         },
-        function (filename, cb) {
-          buildLog('Packing ' + filename + ' into ' + tarPath)
-          pack(filename, tarPath, function (err) {
+        function (filenames, cb) {
+          buildLog('Packing ' + filenames.join(', ') + ' into ' + tarPath)
+          pack(filenames, tarPath, function (err) {
```

Comment:
I'm aware that ^ doesn't really have to do with the PR, but since we're changing this code, why not clean up some stuff as well? :)